scripts/package.py

```diff
@@ -59,7 +59,7 @@ def build_pex_pex(output_file: PurePath, verbosity: int = 0) -> None:
 
 def describe_git_rev() -> str:
     git_describe = subprocess.run(
-        ["git", "describe"], check=True, capture_output=True, encoding="utf-8"
+        ["git", "describe"], check=True, stdout=subprocess.PIPE, encoding="utf-8"
```

Comment:
Testing the packaging and pytest Python 3.6 edge case (`tox -epy36-integration`) caught this bug: `capture_output` was only added in 3.7. CI only runs unit tests for 3.5, ITs start at 3.7.